### ISSUE TITLE
External application using Client class fails to draw circle on the gui.

### ISSUE THREAD

**Issue by Neplik** [TUDelft-CNS-ATM/bluesky#348]
It's no problem to create new aircrafts by client class, but when sending stack command of draw a circle the gui has no change.
```
from bluesky.network.client import Client

class MyClient(Client):
    def __init__(self, actnode_topics=b'', hostname='localhost', event_port=11000, stream_port=11001):
        super().__init__(actnode_topics=actnode_topics)
        self.cmd_sn = 0
        self.connect(hostname=hostname, event_port=event_port, stream_port=stream_port)
        self.update()

    def stack(self, text):
        ''' Stack function to send stack commands to BlueSky. '''
        self.cmd_sn += 1
        self.send_event(b'STACK', text)

    def reset(self):
        self.receive()
        self.stack('reset')

myclient = MyClient(hostname='localhost')
myclient.reset()
myclient.stack(f'CRE CCA1234 A320 49.66 17.12 205 8000 300')
myclient.stack(f'CIRCLE Target 49.66 16.12 5')
```

<img width="626" alt="屏幕截图 2021-12-07 220648" src="https://user-images.githubusercontent.com/50904022/145043755-20bc6a7b-f4fa-4f29-8dc5-667e8efbcfca.png">

**Comment by Neplik** [comment]
I use the newest version of BlueSky and Python version is 3.7.11

**Comment by andersmh** [comment]
I am having the same issue with python 3.8.9. Although, it also includes the drawing of lines. I have attached a traceback.

![Screenshot 2021-12-08 113645](https://user-images.githubusercontent.com/31239471/145193998-e17575c9-ba00-4fbb-881e-c8dd3a9e7268.png)
s.

**Comment by andersmh** [comment]
@Neplik, this can be solved by installing Rtree, as such: `pip install Rtree`

**Comment by jooste** [comment]
actually the problem here (with the external client) is that updates to e.g., the area filter are only sent back to the initiating client, so other clients won't see such changes. This is a bug that needs to be fixed.

**Comment by Neplik** [comment]
> @Neplik, this can be solved by installing Rtree, as such: `pip install Rtree`

Thanks for your advice, but it still can't be fixed.

**Comment by Neplik** [comment]
> actually the problem here (with the external client) is that updates to e.g., the area filter are only sent back to the initiating client, so other clients won't see such changes. This is a bug that needs to be fixed.

Thanks for the reply. Before fixing the bug, I can just write the part of drawing circle in the bluesky.ui.qtgl.guiclient.py .

**Comment by jooste** [comment]
Hi, actually, the bug should be fixed in the latest commit: ce5d480. All simulation state changes that also create a change in the visualisation are now sent to all clients. If you still have issues despite this commit just reopen the issue!
